docs/06-integrations.md

```
@@ -0,0 +1,475 @@
+```python
+# mcp_server.py
+from mcp import Server, Context, Tool
+
+class CompanyMCPServer(Server):
+    def __init__(self):
+        super().__init__()
+        self.register_tool(self.query_database)
+        self.register_tool(self.get_api_docs)
+    
+    @Tool(
+        name="query_database",
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)